### PR TITLE
Log text color styles and empty state

### PR DESCRIPTION
**Pull request by prakashsvmx**
Log text color styles  border, height  and empty state

Closes https://github.com/miniohq/engineering/issues/596

@oscarocastellanos  Please confirm if we still want to add a new font in console for the log text. as of now it is using the existing font.

How does it look?
image

image

image

### CODE REVIEW

**Review by oscarocastellanos — DISMISSED** [review]
LGTM. Thank you  @prakashsvmx

**Review by dvaldivia — DISMISSED** [review]

File: portal-ui/src/screens/Console/Logs/ErrorLogs/LogLine.tsx [hunk 1]
```diff
@@ -62,46 +62,55 @@ const messageForConsoleMsg = (log: LogMessage) => {
   );
 };
 const messageForError = (log: LogMessage) => {
-  const dataStyle = { color: "green" };
+  const dataStyle = {
+    color: "#C83B51",
+    fontWeight: 400,
+    fontFamily: "'monospace'",
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Logs/ErrorLogs/LogLine.tsx:68]: Please remove single quotes, this leads to fallback font

```suggestion
    fontFamily: "monospace",
```
Screen Shot 2022-03-17 at 12 52 50

**Review by bexsoft — CHANGES_REQUESTED** [review]

File: portal-ui/src/screens/Console/Logs/ErrorLogs/LogLine.tsx [hunk 2]
```diff
@@ -62,46 +62,55 @@ const messageForConsoleMsg = (log: LogMessage) => {
...
+    fontSize: "12px",
+  };
+  const labelStyle = {
+    fontFamily: "'monospace'",
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Logs/ErrorLogs/LogLine.tsx:72]: Same case here

**Review by bexsoft — CHANGES_REQUESTED** [review]

File: portal-ui/src/screens/Console/Logs/ErrorLogs/LogLine.tsx (hunk 2 above)
Comment (prakashsvmx) [reply portal-ui/src/screens/Console/Logs/ErrorLogs/LogLine.tsx]: Thank you @bexsoft for pointing this out. :+1:

**Review by bexsoft — APPROVED** [review]
LGTM

**Review by dvaldivia — APPROVED** [review]

**Pull request merged**